Sources/Db/APIs/MySQL.php

```diff
@@ -339,7 +339,7 @@ public function free_result(object $result): bool
 	/**
 	 *
 	 */
-	public function insert(string $method, string $table, array $columns, array $data, array $keys, int $returnmode = 0, ?object $connection = null): int|array|null
+	public function insert(string $method, string $table, array $columns, array $data, array $keys, int $returnmode = DatabaseApi::INSERT_RETURN_MODE_OFF, ?object $connection = null): int|array|null
```

Comment:
Should probably make the same change in the signature for this method in SMF\Db\DatabaseApiInterface.